halo2_proofs/benches/arithmetic.rs

```diff
@@ -1,36 +1,48 @@
 #[macro_use]
```

Comment:
Could you create a separate PR for this benchmark change?

Reply:
removed